modules/installation/con-resource-requirements-orchestrator.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies
+// assembly-install-rhdh-orchestrator-helm.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="con-resource-requirements-orchestrator_{context}"]
+== Resource requirements for Orchestrator
+
+The Orchestrator flavor of {product} comes with the following default resource requests and limits as defined in the Helm chart:
+
+[cols="2,2", options="header"]
+|===
+| Resource
+| Default value
+
+| CPU requests
+| 250m
+
+| Memory requests
+| 64Mi
+
+| CPU limits
+| 500m
+
+| Memory limits
+| 1Gi
+|===
+
+You can customize these values by updating your `values.yaml` file or by passing `--set` flags during installation as shown in the following example :
+
+[source,bash]
+----
+helm install <release_name> redhat-developer/backstage \
```

Comment:
```suggestion
helm upgrade --install <release_name>  openshift-helm-charts/redhat-developer-hub \
```

`helm upgrade --install` is generally recommended, as it will either install a new Helm release or upgrade it with the new parameters.

And similar to my comment [here](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/1264#discussion_r2228879135), we should use the official OpenShift Helm Charts repo.